docs/export-functionality.md

```diff
@@ -2,22 +2,25 @@
 
 ## Export Prices
 
-1. Open Pricing Module;
-1. Select 'Price lists' tab;
-1. Select a specific price list;
-1. Click on 'Prices' widget;
-1. Select prices from the list and click 'Export';
-1. The system will open the 'Prices export' blade;
-1. The next steps are the same as described in https://github.com/VirtoCommerce/vc-module-export
+The Pricing module includes Export functionality, which allows the user to export data using different export criteria. Two types of data objects are allowed to be exported from Pricing module:
+
+1. Price lists;
+1. Price list assignments.
+
+The user can start the data export simply by selecting the data object and going through a very simple and intuitive wizard. The Export function is integrated with the Pricing Module and provided out-of-the-box. The screenshots bellow show the main steps of the data export in Pricing module:
```

Comment:
"The user can start the data export simply by selecting the data object"

I think here we need to add that data filter/selection will be moved to export wizard. Maybe like that:
"The user can start the data export by filtering/selecting data and going to generic export wizard using Export button."

And I would add screenshot when we apply filter for some records, and then show that they are selected in export wizard data tab (like 3 of 15). We could add arrow from filter heading to Select data tab with e.g. 3 of 15 message.